deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -103,13 +124,87 @@ export class NetworkStack extends Stack {
             });
         }
         this.validateVPC(this.vpc)
+        createMigrationStringParameter(this, this.vpc.vpcId, {
```

Comment:
For add-on migrations, will we try to create the same SSM parameter twice and conflict here? Ideally we don't need this for add-on migrations

Reply:
this will create the vpcId ssm parameter with the defaultDeploymentId context which the other stacks will use to import. IIRC defaultDeploymentId != addOnMigrationId so no ssm conflict will occur

Reply:
Synced offline, thanks